Adopt 0 17 (#317)
zcash/zcash-ios-wallet: CHANGELOG.md

# Changelog
## 0.5.0 build 139: Adoption of 0.17.0-beta.rc1

> You may notice that your Transparent receiver has changed! 

Reviewer: ```suggestion
You may notice that your transparent address has changed! 
```
("Receiver" specifically means a component of a UA, so "address" makes more sense here.)

But we haven't forgotten about the previous one.

> We are using UAs now. They contain transparent and Sapling receivers at the moment, 

Reviewer: ```suggestion
We are using Unified Addresses (UAs) now. They contain transparent and Sapling receivers at the moment, 
```

and they will contain other receivers in the future too. Your transparent receiver 
derived from your first UA differs from the one you had previously known. 

> You may wonder, What has happened to it? Have I lost it? Not at all! We implicitly

Reviewer: ```suggestion
You may wonder, What has happened to the old one? Have I lost it? Not at all! We implicitly
```

> You may notice that your Transparent receiver has changed! 
> But we haven't forgotten about the previous one.
> We are using UAs now. They contain transparent and Sapling receivers at the moment, 
> and they will contain other receivers in the future too. Your transparent receiver 
> derived from your first UA differs from the one you had previously known. 
> You may wonder, What has happened to it? Have I lost it? Not at all! We implicitly
> gather the funds and show it to you. Since we are preparing your experience for the Zcash of the near future we'd like you to use this other one transparent receiver instead that shares
> the derivation paths that UAs with Sapling and Orchard will use. Any funds received
> to your old and dear t-address, will remain getting shielded as always :-)

Reviewer 2: ```suggestion
The wallet has been updated to use [Unified Addresses](https://zips.z.cash/zip-0316)
as the primary address format. Unified addresses generated by the wallet currently
contain parts that allow the wallet to receive funds to the Sapling and transparent
pools, and makes these legacy addresses available for if you need to provide an
address to a service that doesn't yet understand the Unified Address format. 

As part of this update, you may notice that the transparent address for your wallet
has changed! The new transparent address you see in your wallet is derived from 
your wallet's unified address, but don't worry; your old address will still continue 
to receive funds and make them available for shielding as normal. 
```

Reviewer: :+1:
